### PR TITLE
[Early-Pull] Grants anyone standing on a crate automatic loud mode because my underlings refuse to listen to me

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Does as the title says (doesn't work on comms)

## Why It's Good For The Game
I can stand on my soapbox

## Proof Of Testing
image

## Changelog

:cl:
add: Standing on a closed crate will give you loudmode in speech
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Teshari player woes

**Comment by Swiftfeather** [comment]
Sorry you don't have windows activated we can't accept a pr from an illegal computer

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
Fix your PR Body

**Comment by theselfish** [comment]
Honestly you could try this upstream. They might take it.

**Comment by projectkepler-ru** [comment]
> Honestly you could try this upstream. They might take it.

if he opens it on skyrat without filling the PR body beside "it's funny" I am still closing it

**Comment by Odairu** [comment]
> Honestly you could try this upstream. They might take it.

https://github.com/tgstation/tgstation/pull/84942 good idea :P

File: code/modules/mob/living/living_say.dm [hunk 1]
```diff
@@ -264,6 +264,15 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 	if(pressure < ONE_ATMOSPHERE*0.4) //Thin air, let's italicise the message
 		spans |= SPAN_ITALICS
 
+	///bubber edit begin - soapbox maxxing
+	if(!(SPAN_COMMAND in spans))
+		var/obj/structure/closet/crate/soapbox = locate(/obj/structure/closet/crate) in T
+		if(soapbox)
+			if(!soapbox.opened && !ismovable(loc))
+				spans |= SPAN_COMMAND
+				message_mods[SAY_FAKE_COMMAND] = TRUE
```
Comment (Majkl-J) [inline code/modules/mob/living/living_say.dm:273]:
```suggestion
		if(soapbox && !soapbox.opened && !ismovable(loc))
			spans |= SPAN_COMMAND
			message_mods[SAY_FAKE_COMMAND] = TRUE
```

Comment (imedial) [reply code/modules/mob/living/living_say.dm]: Fixed

**Comment by StrangeWeirdKitten** [comment]
Will be brought from upstream

**Pull request closed without merging**